content/docs/deployment/ghpages/contents.lr

```diff
@@ -45,14 +45,17 @@ with behavior for GitHub Pages.
 ## CNAME Support
 
 If you want to use a [CNAME :ext](https://en.wikipedia.org/wiki/CNAME) with
-GitHub pages and Lektor you can provide the intended CNAME with the `?cname`
-parameter:
+custom domain configured GitHub pages and Lektor you can provide the intended 
+CNAME with the `?cname` parameter:
 
 ```ini
 [servers.production]
 target = ghpages://your-user/your-repository?cname=www.example.com
 ```
 
+Note that this will overwrite your Github page's custom domain setting
+with given `?cname` parameter on every deployment.
```

Comment:
Please change to:

> Note that this will overwrite whatever custom domain you may have set on GitHub with every deployment.

Note that GitHub has a capital H.